src/m365/planner/commands/task/task-list.spec.ts

```diff
@@ -703,6 +703,27 @@ describe(commands.TASK_LIST, () => {
     });
   });
 
+  it('correctly lists planner tasks with bucketName, deprecated planName, and ownerGroupName', (done) => {
+    const options: any = {
+      debug: false,
+      bucketName: 'Planner Bucket A',
+      planName: 'My Planner Plan',
+      ownerGroupName: 'My Planner Group',
+      verbose: true
+    };
+
+    command.action(logger, { options: options } as any, () => {
+      try {
+        assert(loggerLogSpy.calledWith(taskListResponseBetaValue));
+        done();
+      }
+      catch (e) {
+        done(e);
+      }
+    });
+  });
+
+
```

Comment:
Let's remove this extra white line. While you're at it, the last line of this file is also a white line. Maybe you could remove that as well?